site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,554 @@
+RenderingNG allowed us to ship several very significant "automatic features" that benefit all sites:
+
+- [Site Isolation](/web/updates/2018/07/site-isolation):
+puts cross-origin iframes in different CPU processes,
+for better security and performance isolation.
+- [Vulkan](https://www.vulkan.org/),
+[D3D12](https://docs.microsoft.com/en-us/windows/win32/direct3d12/directx-12-programming-guide), and
+[Metal](https://developer.apple.com/metal/): takes advantage of lower-level APIs that use GPUs more efficiently than OpenGL.
+- More composited animations:
+[SVG](/blog/hardware-accelerated-animations/), background color.
+
+Additional upcoming features unblocked by RenderingNG that we're excited about include:
+
+- [Scroll-linked animations](https://drafts.csswg.org/scroll-animations-1/).
+- [Hidden, yet searchable and accessible DOM](https://github.com/whatwg/html/issues/6040).
+- [Shared element transitions](https://github.com/WICG/shared-element-transitions).
+- [Custom layout](https://www.w3.org/TR/css-layout-api-1/).
+- Off-main-thread compositing; decoupling threading and compositing.
+
+## Key projects that make up RenderingNG
+
+Below is a list of the key projects within RenderingNG. Subsequent blog posts will deep-dive into each of them.
+
+### CompositeAfterPaint
+
+Disentangles compositing from style, layout and paint,
+allowing much-improved reliability and predictable performance, increased throughput,
+and using less memory without sacrificing performance. It began in 2014 and will finish this year.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2015</td>
+      <td>Ship display lists.</td>
+    </tr>
+    <tr>
+      <td>2017</td>
+      <td>Ship new invalidation.</td>
+    </tr>
+    <tr>
+      <td>2018</td>
+      <td>Ship property trees part 1.</td>
+    </tr>
+    <tr>
+      <td>2019</td>
+      <td>Ship property trees part 2.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>Ship SVG.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### LayoutNG
+
+A ground-up rewrite of all layout algorithms,
+for greatly improved reliability and more predictable performance. It began in 2016
+and is planned to finish this year.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2019</td>
+      <td>Ship block flow.</td>
+    </tr>
+    <tr>
+      <td>2020</td>
+      <td>Ship flex, editing.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>Ship everything else.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### BlinkNG
+
+A systematic cleanup and refactoring of the Blink rendering engine into cleanly separated pipeline phases.
+This allows for better caching, higher reliability,
+and re-entrant / delayed-rendering features such as content-visibility and container queries. It began in 2014, and incremental improvement and has been ongoing since. It will complete in 2021.
```

Comment:
```suggestion
and re-entrant or delayed-rendering features such as content-visibility and container queries. It began in 2014, and incremental improvement and has been ongoing since. It will complete in 2021.
```